src/lib.rs

```diff
@@ -79,6 +81,9 @@ impl LightningNode {
             warn!("Object storage is unhealty");
         }
 
+        // Initialize the Transaction Filter
```

Comment:
The step numbering is currently being done according to this [tutorial](https://lightningdevkit.org/tutorials/build_a_node_in_rust/). In there, as initialising a filter is optional, they don't number it. If we number this step, the following steps will be correctly numbered when compared to the tutorial.

What do you mean about it being more than initialising?

Reply:
If we follow the tutorial, then it should be after step 4.
https://lightningdevkit.org/tutorials/build_a_node_in_rust/#optional-initialize-the-transaction-filter.

Reply:
I don't like because it feels like being part of the step `Initialize Persist` which it isn't, as a matter of fact nothing is being persisted here. So I think it is confusing. Even calling it `Step X` could clear that up. Or `Step NaN`. I agree it's tricky if you find it valuable to stick to the tutorial steps.